python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -141,14 +143,16 @@ def wrapper(job, *args, **kwargs):
 
 
 @_MetricsRecovery
-def _process(workspace, dataId, parallelization):
+def _process(pipeline, workspace, dataId, parallelization):
     """Run single-frame processing on a dataset.
 
     Parameters
     ----------
+    pipeline : `lsst.ap.pipe.ApPipeTask`
+        An instance of the AP pipeline.
     workspace : `lsst.ap.verify.workspace.Workspace`
         The abstract location containing input and output repositories.
-    dataId : `str`
+    dataId : `dict` of any
```

Comment:
Oops, that should be "`str` to any".

For me, just saying `dict` adds frustration because I don't know what key/value types the code will assume (and there usually are assumptions).